Creation of HelpGuide.md (#1426)
OWASP/CheatSheetSeries: HelpGuide.md

> # `Help Guide` Cheat Sheet

Reviewer: ```suggestion
# Cheat Sheet Help Guide
```

## Introduction

Welcome to the Cheat Sheet Help Guide. This guide is designed to help you navigate the website and make the most of the features and resources offered. The cheat sheets are packed with valuable information, and we want to ensure you have all the tools you need to use them effectively.

### What's Inside This Guide
1. Basic Navigation
2. Features
3. FAQ
4. Troubleshooting Guide

> 5. Feedback and Support

Reviewer: It'd be nice for these to be anchor links to the referenced sections

Author: Added anchor links to the sections

## Guides

### Basic Navigation
Navigation is easy. Just click on any cheat sheet or cheat sheet series you wish to see on the left side of the site. These links won't go away as you navigate so click away without getting lost.
![Help_Nav](https://github.com/tylersnel/CheatSheetSeries/assets/67352917/020de84c-e18f-477a-acd4-889617666308)

### Features
#### Search
Use the search bar located at the top of the page to search for certain cheat sheets and keywords in cheat sheets. 
![Capture2](https://github.com/tylersnel/CheatSheetSeries/assets/67352917/5af0a995-ef96-42e3-90f6-e7cc8a18cfd1)
#### Dark Mode

> If you prefer a darker theme, there is a dark mode option for the cheat sheets. Click the dark mode button to enable dark mode, which is located next to the search bar.

Reviewer: ```suggestion
If you prefer a darker theme, there is a dark mode option for the cheat sheets. Click the light/dark mode button, located next to the search bar, to switch between modes.
```

![Dark_Mode](https://github.com/tylersnel/CheatSheetSeries/assets/67352917/ce753a58-c34e-4384-a726-e947d851e21f)
#### GitHub Repository
If you wish to visit the project's GitHub repository, click on the repository link next to the search bar.
![Repository_Link](https://github.com/tylersnel/CheatSheetSeries/assets/67352917/8582725d-941f-4d2d-b38d-7986e2659cc6)

### FAQ
Here are some frequently asked questions (FAQs) for the OWASP Cheat Sheet Series:

#### General Information

**Q: What is the OWASP Cheat Sheet Series?**  

> A: The OWASP Cheat Sheet Series is a collection of concise, high-value information on specific application security topics, created by various experts in the field.

Reviewer: ```suggestion
A: The OWASP Cheat Sheet Series is a collection of concise, high-value informational articles on specific application security topics, created by various experts in the field.
```

**Q: Who creates the cheat sheets?**  

> A: The cheat sheets are created by application security professionals who have expertise in the specific topics covered.

Reviewer: ```suggestion
A: The cheat sheets are created by application security professionals who have expertise in the specific topics covered. They are open source and anyone is welcome to contribute by submitting a pull request on the [GitHub repository](https://github.com/OWASP/CheatSheetSeries/).
```

**Q: What topics do the cheat sheets cover?**  
A: The cheat sheets cover a wide range of application security topics, including but not limited to authentication, data validation, secure coding practices, and more.

#### Access and Usage

**Q: How can I access the cheat sheets?**  
A: You can access the cheat sheets directly on the OWASP Cheat Sheet Series website. They are available for free to view and download.

**Q: Can I use the cheat sheets in my projects?**  
A: Yes, the cheat sheets are designed to be practical resources that you can use in your projects to enhance security practices.

**Q: Are the cheat sheets regularly updated?**  
A: Yes, the cheat sheets are periodically reviewed and updated to reflect the latest security practices and emerging threats.

#### Contributions and Community

**Q: Can I contribute to the cheat sheet series?**  

> A: Yes, contributions from the community are welcome. You can contribute by suggesting new topics, providing feedback, or updating existing cheat sheets.

Reviewer: ```suggestion
A: Yes, contributions from the community are welcome. You can contribute by suggesting new topics, providing feedback, or updating existing cheat sheets. Create an issue or pull request on our [GitHub repository](https://github.com/OWASP/CheatSheetSeries/).
```

#### Licensing and Usage Rights

**Q: Under what license are the cheat sheets available?**  
A: Creative Commons Attribution-Share Alike 4.0 International

**Q: Can I use the cheat sheets in my commercial projects?**  
A: Yes, you can use the cheat sheets in commercial projects, provided you adhere to the terms of the open-source license under which they are released.

These FAQs should help users understand the purpose, usage, and contributions related to the OWASP Cheat Sheet Series website.

> ### Troubleshooting Guide

Reviewer 2: I think this section would better be presented as a numbered list or perhaps a bullet list since these are all independent steps.

Author: Good idea I'll update

Author: Added bullet list to the steps
